custom_components/myskoda/climate.py

```diff
@@ -179,33 +190,116 @@ def __init__(self, coordinator: MySkodaDataUpdateCoordinator, vin: str) -> None:
         )
         ClimateEntity.__init__(self)
 
+        self._attr_supported_features = (
+            ClimateEntityFeature.TURN_ON | ClimateEntityFeature.TURN_OFF
+        )
+
+        if self.vehicle.has_capability(
+            CapabilityId.AUXILIARY_HEATING_TEMPERATURE_SETTING
+        ) or self.vehicle.has_capability(
+            CapabilityId.AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY
+        ):
+            self._attr_supported_features |= ClimateEntityFeature.TARGET_TEMPERATURE
+
+    def _has_any_capability(self, cap: list[CapabilityId]) -> bool:
+        """Check if any capabilities in the list is supported."""
+        return any(self.vehicle.has_capability(capability) for capability in cap)
+
+    def _has_capability(self, cap: list[CapabilityId]) -> bool:
+        """Check if all capabilities in the list are supported."""
+        return all(self.vehicle.has_capability(capability) for capability in cap)
```

Comment:
nit: these could probably live in the base `MySkodaEntity` class since they could be useful for other entities too - can always move later though